azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_transport.py

```diff
@@ -322,3 +404,24 @@ def send_event(self, message, callback=None):
 
     def _on_shared_access_string_updated(self):
         self._trig_on_shared_access_string_updated()
+
+    def enable_input_messages(self, callback=None):
+        self._trig_enable_receive(callback)
```

Comment:
You're not setting _subscribe_callback here.  Is this intentional?

Reply:
mistake

---
In reply to: [246548668](https://github.com/Azure/azure-iot-sdk-python-preview/pull/36#discussion_r246548668) [](ancestors = 246548668)